packages/core/src/agent/AgentDependencies.ts

```diff
@@ -0,0 +1,15 @@
+import type { FileSystem } from '../storage/FileSystem'
+import type { EventEmitter as NativeEventEmitter } from 'events'
+import type * as Indy from 'indy-sdk'
+import type fetch from 'node-fetch'
+import type WebSocket from 'ws'
```

Comment:
Maybe you already mention something about thsese two deps. Would it be possible to remove it later? In other words. Just mentioning it in case we forgot it here :)

Reply:
Yes. But we only depend on the @types packages. Not the actual packages themselves